pythonforandroid/recipes/python3/patches/py3.7.1_fix-ctypes-util-find-library.patch

```diff
@@ -1,9 +1,8 @@
-diff --git a/Lib/ctypes/util.py b/Lib/ctypes/util.py
---- a/Lib/ctypes/util.py
-+++ b/Lib/ctypes/util.py
-@@ -67,4 +67,11 @@
-                 return fname
-         return None
+--- Python-3.11.4/Lib/ctypes/util.py	2023-06-07 03:30:27.000000000 +0530
```

Comment:
This whole patch, is only applied on `3.7`, and is not applied on other versions.
Can you revert it?